### PR TITLE
Change helm release process, rename chart

### PR DESCRIPTION
**Pull request by scheibinger**
* I need to rename chart as we have chainlink-cluster repo in the helm registry already. We need to rename crib related one so it won't override existing one.
* Updated workflows for publishing chart to use common github action workflow

### CODE REVIEW

File: .github/workflows/crib-helm-chart-pr.yml [hunk 1]
```diff
@@ -15,23 +15,32 @@ jobs:
...
           gc-org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
+  ci-build-charts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: ci-build-charts
+        uses: smartcontractkit/.github/actions/cicd-build-publish-charts@7ac9af09dda8c553593d2153a975b43b6958fa9f # cicd-build-publish-charts@0.1.3
+        with:
+          # grafana inputs
+          metrics-job-name: ci-build-charts
+          gc-basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
```
Comment (momentmaker) [inline .github/workflows/crib-helm-chart-pr.yml:45]: this should follow the format above:
```
          gc-basic-auth: ${{ secrets.GRAFANA_INTERNAL_BASIC_AUTH }}
          gc-host: ${{ secrets.GRAFANA_INTERNAL_HOST }}
          gc-org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
```

File: .github/workflows/crib-helm-chart-push-develop.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+name: Helm Publish
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - "charts/**"
+      - ".github/workflows/crib-helm-chart.yml"
```
Comment (momentmaker) [inline .github/workflows/crib-helm-chart-push-develop.yml:9]: the new file is called `crib-helm-chart-pr.yml` no?

Comment (scheibinger) [reply .github/workflows/crib-helm-chart-push-develop.yml:9]: yes, good catch 👍

File: .github/workflows/crib-helm-chart-push-develop.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      - ".github/workflows/crib-helm-chart-push-develop.yml"
+jobs:
+  helm_release:
```
[the remark this answers is not in the document]
Comment (scheibinger) [reply .github/workflows/crib-helm-chart-push-develop.yml:12]: ah, that job is a leftover from the previous implementation of helm release, which was pushing a chart to gh-pages. 

If we change how chart is pushed might be worth to update GATI. Thanks for that 🙏

File: .github/workflows/crib-helm-chart-push-develop.yml (hunk 2 above)
Comment (scheibinger) [reply .github/workflows/crib-helm-chart-push-develop.yml]: updated

File: .github/workflows/crib-helm-chart-push-develop.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (scheibinger) [reply .github/workflows/crib-helm-chart-push-develop.yml]: ok, there is nothing in GATI releated to it. we can just remove it. This is already cleaned up in the latest commit.

**Review by momentmaker — APPROVED** [review]

**Review by AnieeG — APPROVED** [review]

**Review by chainchad — APPROVED** [review]

File: .github/workflows/crib-helm-chart-push-develop.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    permissions:
+      id-token: write
+      contents: read
+      actions: read
+    steps:
+      - name: cicd-build-publish-charts
+        uses: smartcontractkit/.github/actions/cicd-build-publish-charts@5b1046c28343660ecb84844c6fa95a66d1cdb52e # cicd-build-publish-charts@0.1.2
+        with:
+          # general inputs
+          charts-dir: chart
```
Comment (momentmaker) [inline .github/workflows/crib-helm-chart-push-develop.yml:34]: should be `charts` no?

File: crib/README.md [hunk 5]
```diff
@@ -2,21 +2,8 @@
...
-If you want to build images you need [docker](https://docs.docker.com/engine/install/) service running
-
-Enter the shell (from the root project dir)
-
-```
-nix develop
-```
-
 # Develop
+<<<<<<<< HEAD:crib/README.md
```
Comment (momentmaker) [inline crib/README.md:6]: merge conflict?

**Pull request closed without merging**